src/debezium-service/templates/config.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{.Values.name}}
+data:
+  DEPLOY_MODE: {{.Values.DEPLOY_MODE}}
+  MEMORY: {{.Values.memory | quote}}
+  GOOGLE_APPLICATION_CREDENTIALS: /opt/harness/svc/cloud_stackdriver.json
+  STACK_DRIVER_LOGGING_ENABLED: {{.Values.STACK_DRIVER_LOGGING_ENABLED | quote}}
+  EVENTS_FRAMEWORK_REDIS_URL: {{.Values.EVENTS_FRAMEWORK_REDIS_URL}}
+  EVENTS_FRAMEWORK_USE_SENTINEL: {{.Values.EVENTS_FRAMEWORK_USE_SENTINEL | quote}}
+  EVENTS_FRAMEWORK_SENTINEL_MASTER_NAME: {{.Values.EVENTS_FRAMEWORK_SENTINEL_MASTER_NAME}}
+  EVENTS_FRAMEWORK_REDIS_SENTINELS: {{.Values.EVENTS_FRAMEWORK_REDIS_SENTINELS}}
```

Comment:
redis related database string needs to go inside deployment.yaml

Reply:
check how the helper functions help reduce your env Vars, also we need to not hardcode this.
Customers set redis credential in only 1 way, via global.database.redis field. Depending on usecase, it can be external and internal deployed redis instance